projects/batfish/src/main/java/org/batfish/smt/Graph.java

```diff
@@ -172,7 +172,7 @@ private void initGraph() {
   }
 
   public static boolean isNullRouted(StaticRoute sr) {
-    return sr.getNextHopInterface() != null && sr.getNextHopInterface().equals("null_interface");
+    return sr.getNextHopInterface().equals("null_interface");
```

Comment:
Can you use constant instead of magic string?

https://github.com/batfish/batfish/blob/f986f43d37b25fab54545720a07ad8a608127fd1/projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/Interface.java#L59

Reply:
Sounds good.